defaultstore.go

```diff
@@ -8,6 +8,11 @@ var (
 	DefaultStore = NewStore()
 )
 
+// Clean the store resources
```

Comment:
```suggestion
// Close the store resources
```

Convention to document a method in Golang suggest to always start with method name.

More info: https://go.dev/blog/godoc